config.json

```
@@ -2714,6 +2714,18 @@
         ],
         "difficulty": 7
       },
+      {
+        "slug": "intergalactic-transmission",
+        "name": "intergalactic-transmission",
+        "uuid": "845c8cf4-207c-438f-9d18-c5e7875ee729",
+        "practices": [
+          "bit-manipulation"
+        ],
+        "prerequisites": [
+          "bit-manipulation"
```

My only suggestion is to add some more prerequisites.

```suggestion
          "bit-manipulation",
          "arrays",
          "exceptions",
          "integral-numbers",
          "for-loops",
          "if-statements"
```

---

Looks good!